src/main/java/ScientificCalculatorSeleniumTest.java

```diff
@@ -0,0 +1,270 @@
+
+        // verify if the result is 5
+        if(result.equals("5")){
+            System.out.println("pass");
+        }
+        else {
+            System.out.println("fail");
+        }
+        driver.close();
+*/
+
+        // open the browser and go to open www.calculator.net'
+// Click on Link Math Calculator
+// Click on Link Scientific Calculator
+//calculate sin 90
+        // how to do it
+        //  Click sin
+        // click 9
+        // click  0
+        // click =
+
+// Read result
+// verify the result is 1.
+// if reset is 1 print pass or print fail.
+
+
+// Reset calculator by click "AC" button
+// calculate tan 45
+// Read result
+// verify the result is 1.
+// if reset is 1 print pass or print fail.
+// Reset calculator by click "AC" button
+// calculate tan 0
+// Read result
+// verify the result is 0.
+// if reset is 0 print pass or print fail.
+// Reset calculator by click "AC" button
+// calculate tan 90
+// Read result
+// verify the result is ∞.
+// if reset is ∞ print pass or print fail.
+        // Reset calculator by click "AC" button
+        WebElement elementresetButton6 =
+                driver.findElement(By.xpath("/html[1]/body[1]/div[3]/div[1]/div[2]/table[1]/tbody[1]/tr[1]/td[1]/div[3]/div[5]/span[3]"));
+        elementresetButton6 .click();
+        WebElement elementtanButton6 =  driver.findElement(By.xpath("//body//div[@id='content']//div//div[2]//div[1]//span[3]"));
+        elementtanButton6.click();
+
+        WebElement elementnineButton3 = driver.findElement(By.xpath("/html[1]/body[1]/div[3]/div[1]/div[2]/table[1]/tbody[1]/tr[1]/td[1]/div[3]/div[1]/span[3]"));
+        elementnineButton3.click();
+
+        WebElement elementzeroButton3 = driver.findElement(By.xpath("/html[1]/body[1]/div[3]/div[1]/div[2]/table[1]/tbody[1]/tr[1]/td[1]/div[3]/div[4]/span[1]"));
+        elementzeroButton3.click();
+
+        WebElement elementisequaltoButton5 = driver.findElement(By.xpath("/html[1]/body[1]/div[3]/div[1]/div[2]/table[1]/tbody[1]/tr[1]/td[1]/div[1]/div[2]"));
+        elementisequaltoButton5 .click();
+// Read result
+        WebElement resultTan90 = driver.findElement(By.xpath("//div[@id='sciOutPut']"));
+        WebElement elementisequaltoButton9 = driver.findElement(By.xpath("/html[1]/body[1]/div[3]/div[1]/div[2]/table[1]/tbody[1]/tr[1]/td[1]/div[1]/div[2]"));
+        elementisequaltoButton5 .click();
+        String result8 = resultTan90.getText();
+        result8 = result.replaceAll("\\s+","");
+        System.out.println("String value of Tan90 is " + result8);
+        //driver.findElement(By.xpath("//div[@id='sciOutPut']"))
+        // verify if the result is 1
+//        if(result8.equals("1")){
+//            System.out.println("pass");
+//        }
+//        else {
+//            System.out.println("fail");
+//        }
+        String valuetan90=Check(result8);
+        System.out.println(valuetan90);
+
+    }
+
+    static  String Check(String results){
```

Comment:
why can nto change the function take  2 inputs.
Like static  String Check(String results, String toVerifyResult) {
  //do only one if condition 
}